styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -194,6 +197,29 @@ private void run() {
           }
           break;
 
+        case RESOURCE:
```

Comment:
Starting to feel the smell of switch inside a switch. Maybe it's better to have a method for subcommand processing.  Can be done via another PR.